docker-resources/docker-compose-base-test-mysql.yml

```diff
@@ -0,0 +1,66 @@
+version: '3.9'
+services:
+    db:
+        image: mysql:8.0-oracle
+        cap_add:
+            - SYS_NICE  # CAP_SYS_NICE
+        command:
+            --information-schema-stats-expiry=0
+            --innodb-flush-method=O_DIRECT
+            --lower-case-table-names=1
+            --sql-mode=NO_ENGINE_SUBSTITUTION
+        #ports:
+           # - "5432:5432" #needs overriding "XXXX:5432" XXXX=your-own-port-per-module-test
+        environment:
+            MYSQL_ROOT_PASSWORD: password
+            MYSQL_DATABASE: valtimo #needs overriding
+            MYSQL_USER: valtimo
+            MYSQL_PASSWORD: password
+# Ports taken
+#   url: jdbc:postgresql://localhost:3301/form-test
```

Comment:
I would not expect jdbc:postgresql urls here. Maybe move the 'Ports taken' somewhere else, or only keep it at the postgres declaration.

Reply:
I changed `postgresql` to `mysql` for now.